v20.2/create-type.md

```diff
@@ -0,0 +1,96 @@
+---
+title: CREATE TYPE
+summary: The CREATE TYPE statement creates a new, enumerated data type in a database.
+toc: true
+---
+
+<span class="version-tag">New in v20.2:</span> The `CREATE TYPE` [statement](sql-statements.html) creates a new, [enumerated data type](enum.html) in a database.
+
+## Required privileges
+
+The user must have the `CREATE` [privilege](authorization.html#assign-privileges) on the parent database.
+
+## Synopsis
+
+<div>
+  {% include {{ page.version.version }}/sql/diagrams/create_type.html %}
+</div>
+
+## Parameters
+
+Parameter | Description
+----------|------------
+`type_name` | The name of the type.
+`opt_enum_val_list` | A list of values that make up the type's enumerated set.
+
+## Example
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE TYPE status AS ENUM ('open', 'closed', 'inactive');
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW ENUMS;
+~~~
+
+~~~
+   name  |        value
+---------+-----------------------
+  status | open|closed|inactive
+(1 row)
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE TABLE accounts (
```

Comment:
cc @RichardJCai 

I think we want to mention that a user must have `USAGE` privilege in order to add it to a table like this

Reply:
bump